series/static/series/subscription.js

```diff
@@ -0,0 +1,107 @@
+/*
+We need to cache locally if the user is subscribed or not so he can
+toggle his subscription without reloading the page every time
+*/
```

Comment:
Have you tried Object Oriented Javascript? JS has classes too, now.

```javascript
class MyFoo {
   constructor(foo) {
      this._foo = foo;
   }
   show() {
      console.log(this._foo);
   }
}
```
I think it would make this code more maintainable (and please the teachers :p). We can also agree that we'll refactor this later — let me know what you think.